Swift/QtUI/QtXMLSenderWidget.cpp

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2010-2016 Isode Limited.
+ * All rights reserved.
+ * See the COPYING file for more information.
+ */
```

Comment:
If this file is completely new and written by yourself, it should have a copyright header with your name on it. See for example how another student has done it in their PR https://github.com/swift/swift/pull/46/files .